### DISCUSSION TITLE
Payment issue

### DISCUSSION THREAD

**Discussion by Tericdevelopers** [community/community#60574] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

Hello team iam unable to renew my co-pilot account. It is saying update payment details. can you help me with this.Also it shows your  account for purchase is locked.

**Comment by davecheney** [comment, marked as answer]
@Tericdevelopers i'm sorry you are having difficulties. Unfortunately this is not the correct forum for assistance with payment issues. Please contact support https://support.github.com/contact.

**Comment by Tericdevelopers** [comment]
We tried to contact  your support team for billing. But I don't get
any response for your side.

**Reply by davecheney** [reply]
I’m sorry for the delay, support are the ones who can resolve this, I’m sure they will get to your reply in sequence.
